DESCRIPTION

```diff
@@ -21,12 +22,20 @@ Imports:
     usethis,
     withr
 Suggests:
+    bookdown,
```

Comment:
FWIW the distinction I use for imports vs suggests for books is that imports is need to run the code in the book; suggests is needed to build the book itself